### ISSUE TITLE
Implement new "Segmentation Visualizer" for visualizing Primary GIE segmentation results

### ISSUE THREAD

**Issue by rjhowell44** [prominenceai/deepstream-services-library#526]
New component for visualizing the PGIE's segmentation results.  The uniquely named component is created with Output dimensions. 
The `batch_size` and `gpu_id` attributes are controlled by the Parent PGIE once added.

Segmentation Visualizer API and return codes.

```C
#define DSL_RESULT_SEGVISUAL_NAME_NOT_UNIQUE                        0x00600001
#define DSL_RESULT_SEGVISUAL_NAME_NOT_FOUND                         0x00600002
#define DSL_RESULT_SEGVISUAL_THREW_EXCEPTION                        0x00600003
#define DSL_RESULT_SEGVISUAL_IN_USE                                 0x00600004
#define DSL_RESULT_SEGVISUAL_SET_FAILED                             0x00600005
#define DSL_RESULT_SEGVISUAL_PARAMETER_INVALID                      0x00600006

/**
 * @brief Creates a new, uniquely named Segmentation Visualizer. 
 * @param[in] name unique name for the new Segmentation Visualizer
 * @param[in] width output width in pixels, typically same as input to GIE
 * @param[in] height output height in pixels, typically same as input to GIE
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_SIGVISUAL_RESULT
 */
DslReturnType dsl_segvisual_new(const wchar_t* name, uint width, uint height);

/**
 * @brief Returns the output dimensions, width and height, for the named
 * Segmentation Visualizer.  
 * @param[in] name name of the Segmentation Visualizer to query
 * @param[out] width current output width in pixels
 * @param[out] height current output height in pixels
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_SIGVISUAL_RESULT
 */
DslReturnType dsl_segvisual_dimensions_get(const wchar_t* name, 
    uint* width, uint* height);

/**
 * @brief Sets the output dimensions, width and height, for the named 
 * Segmentation Visualizer.
 * @param[in] name name of the Display to update
 * @param[in] width new output width in pixels
 * @param[in] height new output height in pixels
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_SIGVISUAL_RESULT
 */
DslReturnType dsl_segvisual_dimensions_set(const wchar_t* name, 
    uint width, uint height);
```

**Comment by rjhowell44** [comment]
Merged into v0.16.alpha - still needs docs

**Comment by rjhowell44** [comment]
Examples and docs merged into v0.16.alpha
